src/test/Test/Utilities.hs

```diff
@@ -6,50 +6,69 @@
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
 
--- | Utility types used purely for testing.
+-- | Utility functions and types used purely for testing.
 --
 -- Copyright: © 2018-2020 IOHK
 -- License: Apache-2.0
 --
-module Test.Cardano.Types
+module Test.Utilities
```

Comment:
it is quite Cardano specific, I would convey `Cardano` in namescope or at least underline it in comments

Reply:
Fixed here: 7fe4b2394c7f9c98f38b3f1ae4e0eb3f273f3475